adot-testbed/app/src/test/java/software/amazon/adot/testbed/LogsTests.java

```diff
@@ -96,7 +97,7 @@ private GenericContainer<?> createAndStartCollector(String configFilePath, Strin
     void testSyslog() throws Exception {
         String logStreamName = "rfcsyslog-logstream-" + uniqueID;
         collector = createAndStartCollector("/configurations/config-rfcsyslog.yaml", logStreamName);
-
+        Thread.sleep(50000);
```

Comment:
Why is this sleep required? And why is to so long? If this is really required then I would expect a comment describing its necessity.

Reply:
It is really not required, please ignore, removed it